app/config/FeatureSwitches.scala

```diff
@@ -30,6 +30,7 @@ trait FeatureSwitches {
   def isPassDeleteIntentEnabled: Boolean
   def isTemporalValidationEnabled(implicit request: Request[_]): Boolean
   def isOpwEnabled: Boolean
+  def isDesIf_MigrationEnabled: Boolean
```

Comment:
Can we rename this variable to **isDesIfMigrationEnabled** to keep it consistent with other feature switch names? In the app-configs we can still keep it as **desIf_Migration**